odra-vm/src/vm/odra_vm_state.rs

```diff
@@ -74,6 +74,7 @@ impl OdraVmState {
 
     pub fn emit_event(&mut self, event_data: &Bytes) {
         let contract_address = self.callstack.current().address();
+        #[allow(clippy::manual_inspect)]
```

Comment:
Why?

Reply:
Newest clippy wants me to rewrite the code using inspect, but I failed...